healthcare/setup.py

```diff
@@ -1,9 +1,78 @@
-from __future__ import unicode_literals
 import frappe
-
 from frappe import _
 from erpnext.setup.utils import insert_record
 
+
+data = {
+	'desktop_icons': [
+		'Patient',
+		'Patient Appointment',
+		'Patient Encounter',
+		'Lab Test',
+		'Healthcare',
+		'Vital Signs',
+		'Clinical Procedure',
+		'Inpatient Record',
+		'Accounts',
+		'Buying',
+		'Stock',
+		'HR',
+		'ToDo'
+	],
+	'default_portal_role': 'Patient',
+	'restricted_roles': [
+		'Healthcare Administrator',
+		'LabTest Approver',
+		'Laboratory User',
+		'Nursing User',
+		'Physician',
+		'Patient'
+	],
+	'custom_fields': {
+		'Sales Invoice': [
+			{
+				'fieldname': 'patient', 'label': 'Patient', 'fieldtype': 'Link', 'options': 'Patient',
+				'insert_after': 'naming_series'
+			},
+			{
+				'fieldname': 'patient_name', 'label': 'Patient Name', 'fieldtype': 'Data', 'fetch_from': 'patient.patient_name',
+				'insert_after': 'patient', 'read_only': True
+			},
+			{
+				'fieldname': 'ref_practitioner', 'label': 'Referring Practitioner', 'fieldtype': 'Link', 'options': 'Healthcare Practitioner',
+				'insert_after': 'customer'
+			}
```

Comment:
Adding Inpatient Record Link to Sales Invoice is also a good idea. The IP Record on sales invoice is required for insurance claims and it also helps to track patient billing during that particular episode of admission.